js/apps/system/_admin/aardvark/APP/frontend/js/collections/foxxCollection.js

```diff
@@ -28,76 +28,21 @@
       this.sortOptions.desc = val;
     },
 
-    // Install Foxx from github repo
-    // info is expected to contain: "url" and "version"
-    installFromGithub: function (info, mount, callback, isLegacy, flag) {
-      var url = arangoHelper.databaseUrl('/_admin/aardvark/foxxes/git?mount=' + encodeURIComponent(mount));
-      if (isLegacy) {
+    install: function (mode, info, mount, options, callback) {
```

Comment:
These methods all did the same thing except using slightly different URLs. We now pass the bit in the URL that changes in the arguments as "mode".